src/bundle/Resources/public/js/scripts/core/multilevel.popup.menu.js

```diff
@@ -0,0 +1,385 @@
+(function (global, doc, ibexa, Popper) {
+    class MultilevelPopupMenu {
+        constructor(config) {
+            this.container = config.container;
+            this.triggerElement = config.triggerElement;
+            this.referenceElement = config.referenceElement ?? this.triggerElement;
+            this.onTopBranchClosed = config.onTopBranchClosed ?? (() => {});
+            this.onTopBranchOpened = config.onTopBranchOpened ?? (() => {});
+            this.processBranchOnInitAfter = config.processBranchOnInitAfter ?? (() => {});
+            this.processItemOnInitAfter = config.processItemOnInitAfter ?? (() => {});
+            this.initialBranchPlacement = config.initialBranchPlacement ?? 'right-start';
+            this.initialBranchFallbackPlacements = config.initialBranchFallbackPlacements ?? ['right-end', 'left-start', 'left-end'];
+
+            this.hoveredItemsBranches = new Set();
+            this.hoveredBranches = new Set();
+
+            this.handleClickOutside = this.handleClickOutside.bind(this);
+            this.handleItemWithSubitemsClick = this.handleItemWithSubitemsClick.bind(this);
+
+            doc.addEventListener('click', this.handleClickOutside, false);
```

Comment:
shouldn't it be in init?

Reply:
I moved it here in case of someone has custom init.